bin/setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+echo ">> Install python requirements from 'requirements-dev.txt'"
+pip install -r requirements-dev.txt > /dev/null
```

Comment:
setup.sh is in bin and requirements-dev.txt is outside bin so this command will not get the requirements-dev.txt.
And I think its fine if we avoid > /dev/null because there are cases when some packages do not get install due to some library missing at the system level. If user can see the error then it will be helpful for them.